tsconfig.json

```diff
@@ -8,6 +8,8 @@
     "noImplicitAny": true,
     "declaration": true,
     "isolatedModules": true,
+    "moduleResolution": "node",
+    "target": "es2019",
```

Comment:
Needed for `flatMap()` and `flat()`. Note that these `tsconfig` fields were already set in `tsconfig.build.json`.